Brickwall/models.py

```diff
@@ -0,0 +1,67 @@
+from sqlalchemy import Column, Integer, String, DateTime, Boolean, ForeignKey
+from . import db
+from enum import Enum
+
+
+
+class Review(db.Model):
+    __tablename__ = "review"
+    review_id = Column(Integer, primary_key=True)
+    # user name of reviewer
+    member = Column(String)
+    # whether the person worked full time or co-op
+    # takes a string, but precondition is it should take one of two strings ("full-time", "co-op")
+    type = Column(String)
+    company_id = Column(ForeignKey('company.company_id'))
```

Comment:
You should state whether or not each column is nullable. Basically required vs. optional fields
Example: https://github.com/ComputerScienceHouse/packet/blob/develop/packet/models.py#L136